### DISCUSSION TITLE
Downloading maps on user behalf

### DISCUSSION THREAD

**Discussion by leryss** [ppy/osu#13472] [Q&A] [answered]
Recently I've began to implement a beatmap searcher with some filters that the standard search on the official website doesnt have (http://osu.leryss.me/). My question is if it is possible to implement a download button similar to the one on the official website ? 

Maybe a CORS request to the download API with a user token obtained through OAuth ? Or is such a feature only allowed on the official website ?

**Comment by ribbanya** [comment, marked as answer]
https://github.com/ppy/osu-web/issues/5913#issuecomment-613744409

---

also [this](https://www.reddit.com/r/osugame/comments/gb1oy9/so_apparently_the_best_massdownloading_tool_for/fp3wa9o?utm_source=share&utm_medium=web2x&context=3)

**Comment by bdach** [comment]
This is the wrong repository to ask this on in general.

**Reply by frenzibyte** [reply]
Yep, this should've gone to https://github.com/ppy/osu-web at least. This repository is for the osu!lazer client, completely irrelevant repo from what you're asking for here.

**Reply by leryss** [reply]
Didnt find any discussions tab in osu-web and I didnt want to add a issue just for a question
